etc/config/sway.amazon.properties

```
@@ -0,0 +1,19 @@
+compilers=&sway
+defaultCompiler=swayv0667
+objdumper=/opt/compiler-explorer/gcc-14.1.0/bin/objdump
```

We've since installed gcc-14.2.0; I suspect we have a few places referring to this old place, so apologies but can you update to 14.2.0?

```suggestion
objdumper=/opt/compiler-explorer/gcc-14.2.0/bin/objdump
```